mne_bids_pipeline/_run.py

```diff
@@ -361,12 +366,17 @@ def _get_step_path(
 ) -> pathlib.Path:
     if stack is None:
         stack = inspect.stack()
+    paths = list()
     for frame in stack:
         fname = pathlib.Path(frame.filename)
         if "steps" in fname.parts:
             return fname
+        else:  # pragma: no cover
```

Comment:
... and this can happen when you are running in parallel (e.g. with `n_jobs=4`). The `__mne_bids_pipeline_step__` var will be used.